packages/create-waku/src/helpers/examples.ts

```diff
@@ -19,6 +19,14 @@ export async function isUrlOk(url: string): Promise<boolean> {
   }
 }
 
+/**
+ * this is a part of the response type for github "Get a repository" API
+ * @see {@link https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository|GitHub REST API}
+ */
+interface GetRepoInfo {
+  /** A default branch of the repository */
+  default_branch: string;
+}
```

Comment:
I take a part of the response type from github docs [here](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository)

cc. @himself65